### PR TITLE
Add mention of Dash Investment Foundation

### PR DESCRIPTION
**Pull request by Sidem**
remove outdated links and email
add mention of DIF and possibility of providing equity to the network

### CODE REVIEW

File: governance/eight-steps.rst [hunk 1]
```diff
@@ -92,10 +92,8 @@ Keep the community updated when your proposal passes
 Consider arrangements for large requests
   If you are requesting a significant amount of funding, there is an
   understandable concern that you will deliver on your promises to the
-  network. Reach out to trusted intermediaries such as `Dash Core Group,
-  Inc. <bradley.zastrow@dash.org>`_ or `Green Candle, Inc.
-  <https://greencandle.io>`_ in advance for advice on escrow, and make
-  the conditions for escrow release public and part of the proposal. If
+  network. Reach out to the `Dash Investment foundation <https://www.dashinvests.org/>` in advance for advice on escrow or providing equity, and make
```
Comment (strophy) [inline governance/eight-steps.rst:95]: Please add a double underscore `__` after the URL to create an anonymous hyperlink and add line breaks after 78 chars. Also `foundation` should be capitalised.

Comment (Sidem) [reply governance/eight-steps.rst]: Done

**Review by strophy — APPROVED** [review]
:+1: LGTM!

**Comment by thephez** [comment]
@Sidem Can you rebase this onto 0.17.0? Or I can just open a separate PR from my copy of your branch where I've done this (https://github.com/thephez/dash-user-docs/tree/Sidem-patch-4).

**Comment by Sidem** [comment]
> @Sidem Can you rebase this onto 0.17.0? Or I can just open a separate PR from my copy of your branch where I've done this (https://github.com/thephez/dash-user-docs/tree/Sidem-patch-4).

Sounds good to me, just go ahead

**Comment by thephez** [comment]
Closing in favor of #92

**Pull request closed without merging**